src/main/java/dev/codesupport/web/common/security/access/AccessEvaluatorFactory.java

```diff
@@ -24,15 +25,29 @@
      */
     @Autowired
     public AccessEvaluatorFactory(ApplicationContext context) {
-        evaluatorMap = new HashMap<>();
+        this.context = context;
+    }
+
+    /**
+     * Lazy loads the evaluatorMap
+     * <p>Upon first request for an evaluator, loads the evaluators from the applciation context.
+     * This is done to combat a bean load order issue that surfaced.</p>
+     *
+     * @return The map of evaluators
+     */
+    Map<String, AbstractAccessEvaluator<?>> getEvaluatorMap() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5197432)